libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -97,17 +107,17 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
   ref,
 ) {
   const as: ElementType = multiline ? 'textarea' : 'input'
-  const inputVariant = tokens[variant]
-  const spacings = tokens.spacings.comfortable
+  const inputVariant = input[variant]
+  const spacings = tokens.comfortable.spacings
```

Comment:
Not sure why we had `compact` in tokens, too. Doesn't seem like any other component uses Input tokens.. But I left it since we'll work on compact next .

Reply:
Think it was just some leftovers from internal testing 🙈